pkg/operation/seed/seed.go

```diff
@@ -53,6 +53,68 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 )
 
+// NewBuilder returns a new Builder.
+func NewBuilder() *Builder {
+	return &Builder{
+		seedObjectFunc: func() (*gardencorev1beta1.Seed, error) { return nil, fmt.Errorf("seed object is required but not set") },
+	}
+}
+
+// WithSeedObject sets the seedObject attribute at the Builder.
```

Comment:
Can you check the doc strings of all the introduced builders, please? There are some occurrences stating that the method sets the `*Func` attribute of the builder and others stating it sets the attribute without the `Func` suffix (I guess, this is just a leftover)
```suggestion
// WithSeedObject sets the seedObjectFunc attribute at the Builder to return the given Seed.
```